### PR TITLE
test(refactor): moved env variables to collector package

### PR DESCRIPTION
**Pull request by kirrg001**
no issue

Reasons:
- env variables had to be declared in 2 places
- `cd packages/collector && npm run test` fails if you havent set the env variables manually in shell
- simply moved them to an init file for the collector test env for now (could use dotenv later if there are more use cases)

### CODE REVIEW

File: packages/collector/test/initEnv.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/*
```
Comment (willianpc) [inline packages/collector/test/initEnv.js:1]: +1. for DRYing stuff <3

File: .circleci/config.yml [hunk 2]
```diff
@@ -21,6 +21,10 @@ shared: &shared
     - POSTGRES_PASSWORD=nodepw
     - POSTGRES_DB=nodedb
     - MEMCACHED=localhost:11211
+    - MSSQL_HOST=localhost
+    - MSSQL_PORT=1433
+    - MSSQL_USER=sa
+    - MSSQL_PW=stanCanHazMsSQL1
```
Comment (kirrg001) [inline .circleci/config.yml:27]: These were missing at all in the circle ci config :)

File: .circleci/config.yml [hunk 3]
```diff
@@ -86,7 +90,7 @@ shared: &shared
     # For more recent Node.js versions (>= 10) we use the latest 1.x as stated in package.json. We can't use grcp@1.10.1
     # in Node.js 10 since it fails to build under Node.js >= 10. We still want to explicitly test the combination of
     # Node.js 8 with grpc@1.10.1 for reasons.
-    - run: '([[ $(node -v) =~ ^v8.*$ ]] && npm install grpc@1.10.1) || true'
+    - run: "([[ $(node -v) =~ ^v8.*$ ]] && npm install grpc@1.10.1) || true"
```
Comment (kirrg001) [inline .circleci/config.yml:93]: Consistency styling update

**Review by basti1302 — APPROVED** [review]
LGTM

File: .circleci/config.yml (hunk 2 above)
Comment (basti1302) [reply .circleci/config.yml:27]: Probably not required because the default values work for CircleCI, but +1 to add them here as well for consistency. 👍

**Review by basti1302 — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (kirrg001) [reply .circleci/config.yml:27]: FYI Tests failed in CircleCi because of the missing envs :)

Comment (basti1302) [reply .circleci/config.yml:27]: Ah, right! Now that we set them in the code if they are not present, they are absolutely required here. Didn't think that through properly 👍

**Pull request merged**